bridges/snowbridge/primitives/router/src/outbound/tests.rs

```diff
@@ -631,15 +631,15 @@ fn xcm_converter_with_fees_greater_than_reserve_fails() {
 }
 
 #[test]
-fn xcm_converter_convert_with_empty_xcm_yields_unexpected_end_of_xcm() {
+fn xcm_converter_convert_with_empty_xcm_yields_unexpected_instruction() {
 	let network = BridgedNetwork::get();
 
 	let message: Xcm<()> = vec![].into();
 
 	let mut converter = XcmConverter::new(&message, &network);
 
 	let result = converter.convert();
-	assert_eq!(result.err(), Some(XcmConverterError::UnexpectedEndOfXcm));
+	assert_eq!(result.err(), Some(XcmConverterError::UnexpectedInstruction));
```

Comment:
My comment above will make this modification unnecessary.

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/116/commits/e8c9324ec4b9f93ce9594644a35257d979f83b3a